src/ListBox/ListBox.stories.jsx

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable react/no-multi-comp */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import centered from '@storybook/addon-centered';
+import 'bootstrap/scss/utilities/_screenreaders.scss';
+
+import Button from '../Button';
+import ListBox from '../ListBox';
+import ListBoxOption from '../ListBoxOption';
+
+class ListBoxWrapperForOnSelect extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.onSelect = this.onSelect.bind(this);
+
+    this.state = {
+      selectedOption: null,
+      selectedOptionIndex: null,
+    };
+
+    this.fruits = {
+      Apple: '🍎',
+      Orange: '🍊',
+      Strawberry: '🍓',
+      Banana: '🍌',
+    };
+  }
+
+  onSelect(option, index) {
+    this.setState({
+      selectedOption: option,
+      selectedOptionIndex: index,
+    });
+  }
+
+  getSelectedFruitEmoji(fruit) {
+    return fruit ? this.fruits[fruit] : '';
+  }
+
+  render() {
+    const children = Object.keys(this.fruits).map((fruit, index) => (
+      <ListBoxOption
+        key={fruit}
+        onSelect={() => this.onSelect(fruit, index)}
+        style={{ textAlign: 'center' }}
+      >
+        {fruit}
+      </ListBoxOption>
+    ));
+
+    return (
+      <React.Fragment>
+        <span aria-live="polite">
+          Selected Fruit:
+          {this.state.selectedOptionIndex !== undefined ?
```

Comment:
Nit: I often find it's easier to read in the positive. In this case, you could flip the conditional to be `this.state.selectedOptionIndex === undefined` rather.